nx-X11/lib/src/XlibInt.c

```diff
@@ -2921,15 +2926,12 @@ void _XEnq(
 		/* If dpy->qfree is non-NULL do this, else malloc a new one. */
 		dpy->qfree = qelt->next;
 	}
-        else if ((qelt = Xmalloc(sizeof(_XQEvent))) == NULL) {
+	else if ((qelt = Xmalloc(sizeof(_XQEvent))) == NULL) {
 		/* Malloc call failed! */
 		ESET(ENOMEM);
-#ifdef NX_TRANS_SOCKET
                 _XIOError(dpy);
-
+#ifdef NX_TRANS_SOCKET
```

Comment:
This doesn't look correct here... check the ifdef NX_TRANS_SOCKET block once more.

Reply:
Before:
```
#ifdef NX_TRANS_SOCKET
                _XIOError(dpy);
                return;
#else
                _XIOError(dpy);
#endif
```

Now:
```
                 _XIOError(dpy);
+#ifdef NX_TRANS_SOCKET
                 return;
 #endif
```

I'd call that equivalent, right?

Reply:
Ok. You are right.